tests/ember_debug/object-inspector-test.js

```diff
@@ -296,12 +301,14 @@ module('Ember Debug - Object Inspector', function(hooks) {
 
     inspected.set('name', 'Alex');
 
+    await new Promise(res => setTimeout(res, 400));
```

Comment:
Should we try something like `await settled()` here?

Reply:
At the least abstract this to be a separate helper I think